Dockerfile

```diff
@@ -76,7 +76,7 @@ RUN \
     && chown -R ${UID}:${GID} /etc/s6 /run /srv/* /var/lib/nginx /var/www \
     && chmod o+rwx /run /var/lib/nginx /var/lib/nginx/tmp \
 # Clean up
-    && rm -rf "${GNUPGHOME}" /tmp/* \
+    && rm -rf /tmp/* \
```

Comment:
As an alternative solution (if it is really some expansion problem), could not we just delete `"${GNUPGHOME}"` instead?
After all, is there even any other temp dir in there except of the one we just created?

Are tmpfs mounts even included in Docker images BTW? If not, the cleanup may actually not really be needed (though I would keep the `GNUPGHOME` home at least as it is just a clean way/code of doing things.

Reply:
1. This change affects the Dockerfile, which applies at build-time of the container image. The container runtime will use an overlay fs, LVM thin pool volume, aufs or other such mechanism to capture any changes done in that stage of the build. tmpfs may get used for the VOLUMES at run-time, depending on your container runtime and configuration of the container:
https://github.com/PrivateBin/docker-nginx-fpm-alpine/blob/fa2d71bd8a2d5df4fad0d46c3bba9440c8ee3cb0/Dockerfile#L88-L89
2. We do use /tmp (and only that path) extensively during the container build to store, you guessed it, temporary files. It is where the git repo gets cloned into or the tar-ball and signature downloaded to. Composer and Alpines apk are also using it and may not always fully clean up after themselves.
https://github.com/PrivateBin/docker-nginx-fpm-alpine/blob/fa2d71bd8a2d5df4fad0d46c3bba9440c8ee3cb0/Dockerfile#L41-L58
All of these leftovers should be cleaned up before the end of that particular image layer, so we don't include these in the image we produce (smaller image size, hence faster startup and less attack surface).